soroban/README.md

```diff
@@ -0,0 +1,62 @@
+## Deploy all contracts
+
+One time deployment instructions. If you need to make changes to a previously deployed contract, upgrade it instead (see below).
```

Comment:
Can you make more information of ${MY_ACCOUNT} and how to manage the private key of the account?